microsoft-edge/toc.yml

```diff
@@ -1336,6 +1336,11 @@
           href: web-platform/password-reveal.md
           displayName: 
 
+        - name: Use region information in your web content
```

Comment:
```suggestion
        - name: OS Regional Data Display in Microsoft Edge
```

Reply:
@JasonAndrewWriter - if you change here, change in usual list of spots:
title: yaml field
h1 title
link text in articles that link here (probably 0)